f5_openstack_agent/lbaasv2/drivers/bigip/resource_manager.py

```diff
@@ -254,27 +254,39 @@ def _create_bwc_payload(self, loadbalancer, bandwidth):
             bwc_policy['maxRate'] = bandwidth * 1000 * 1000
         return bwc_policy
 
-    def __get_bwc_policy_name(self, loadbalancer):
+    @staticmethod
```

Comment:
Must it be a static method?

Reply:
These three functions get_bwc_policy_name, get_bandwidth_value and get_bwc_irule_name are needed  both by   Class Loadbalancer and Listener,  in order to avoid the duplicated declaration in both these two classes, I changed the definition of these three function to staticmethod in class Loadbalancer. So that we could refer these three functions in Listener class by LoadBalancerManager.get_*....